userprofile/serializers.py

```diff
@@ -4,13 +4,20 @@
 class UserProfileSerializer(serializers.ModelSerializer):
     detail_url = serializers.CharField(read_only=True)
 
+
     class Meta:
         model = UserProfile
         fields = '__all__'
 
+    user_type = serializers.CharField(source='get_usertype_display', required=False)
```

Comment:
@regilkent123 @iamkatrinaanne why is there `user_type` and `usertype`?

Reply:
@ozknightwalker if ill put source at the usertype, post will not initiate

Reply:
post will not execute diay